decentralized-message-queue/src/DMQ/NodeToClient.hs

```diff
@@ -0,0 +1,135 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric  #-}
+{-# LANGUAGE LambdaCase     #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+module DMQ.NodeToClient where
```

Comment:
Did you also included the tests for node-to-client & node-to-node codecs & the handshake?

Reply:
I did not, I'll add that to my TODO issue